canvas_modules/common-canvas/src/common-properties/components/virtualized-table/virtualized-table.jsx

```diff
@@ -465,6 +465,7 @@ class VirtualizedTable extends React.Component {
 		// It does a direct DOM manipulation to the parent, outside React's VirtualDOM.
 		// Since the actual DOM is not available when unit testing, we are passing in a default
 		// width of 500 and a default height of 300.
+		const tableHeight = this.props.tableHeight || defaultTestHeight;
```

Comment:
With these changes, do we still need a `defaultTestHeight`?

Reply:
i think yes, the old logic `height ? height : defaultTestHeight` would work off of the Table calculating the height, but now we calculate the height and data.length can be 0 so it will force a height 0 which may affect testing. This is not something I'm 100% sure on, this would leave it in

Reply:
> but now we calculate the height and data.length can be 0 so it will force a height 0

Since that is what the code does then the test should also behave the same. The `defaultTestHeight` was introduced because AutoSizer requires the DOM to exist to fit itself into and will return a `height` to set in the style. But because jest uses the virtual DOM, there was no parent container to fit itself into. Now that a `height` is being calculated, we won't ever need to read the `height` from AutoSizer, which in turn makes me believe its safe to remove the `defaultTestHeight` but will need to run the tests to make sure

Reply:
ok, I'm comfortable to proceed with removal if you are aligned. Will test additional ways.